app/views/users/_form.html.erb

```diff
@@ -20,8 +20,8 @@
 
   <div class='tab-content'>
     <div class='tab-pane active' id='primary'>
-
-      <%= text_f f, :login, :disabled => f.object.login == 'admin' %>
+      <%  msg =  @user.internal? ? "": "External user login can not be changed" %>
```

Comment:
Seems like this introduced a regression when creating a new user http://projects.theforeman.org/issues/21849